lib/common/common_helper.rb

```diff
@@ -1,25 +1,29 @@
 # encoding: UTF-8
 
+# Location directories
 LIB_DIR              = File.expand_path(File.join(__dir__, '..')) # wpscan/lib/
 ROOT_DIR             = File.expand_path(File.join(LIB_DIR, '..')) # wpscan/ - expand_path is used to get "wpscan/" instead of "wpscan/lib/../"
 USER_DIR             = File.expand_path(Dir.home) # ~/
+
+# Core WPScan directories
 CACHE_DIR            = File.join(USER_DIR, '.wpscan/cache') # ~/.wpscan/cache/
 DATA_DIR             = File.join(USER_DIR, '.wpscan/data') # ~/.wpscan/data/
-CONF_DIR             = File.join(ROOT_DIR, 'conf') # wpscan/conf/
+CONF_DIR             = File.join(ROOT_DIR, '.wpscan/conf') # ~/.wpscan/conf/
```

Comment:
I think the path is missing the ```~```? As it currently is, wouldn't this create a hidden directory within the wpscan directory, rather than the user's home folder?

Reply:
I agree!